.github/workflows/ci.yaml

```diff
@@ -11,7 +11,9 @@ on:
 
 env:
   ROS_DISTRO: galactic
-  PACKAGES_TO_TEST: "uwrt_mars_rover"
+  PACKAGES_TO_TEST: >
+    uwrt_mars_rover_drivetrain_description
+    uwrt_mars_rover_drivetrain_hw
```

Comment:
fyi @niiquaye: 

We need to add every new package we want tested into this list. I added a thing into the pr template to address this

Reply:
alright sounds good